package.json

```diff
@@ -15,6 +15,7 @@
   },
   "dependencies": {
     "color": "^3.1.0",
+    "react-native-svg": "^9.5.3",
```

Comment:
Yeah I left out the instructions because I wasn't sure how it should be added to the project. I think as pear is necessary, because then `npm` will show a warning when the project is installed.